CHANGELOG.md

```diff
@@ -36,6 +36,10 @@
 
     *Kate Higa*
 
+* Update the `Truncate` component to accept `:strong` as a tag.
+
+    *Amélia Chavot*
+
```

Comment:
I think this could be in the `Updates` section! :)

Reply:
To the update section then!

Do we have some rules for where changes should be written down? Or do we do it by intuition like I did? ^^'

Reply:
@Artemisia-Absinthium good question! Would you be up for updating our contributing guidelines to mention these sections?

Reply:
Yep! I will draft a new PR for that. :)